src/backend/devices/kpro/kpro.py

```diff
@@ -74,44 +82,77 @@ def _establish_connection(device):
         )
         return entry_point
 
+    """
+    Note on sending messages to the S300 - Not sure if this is something
```

Comment:
cool, thanks for the note, we will have to investigate what the hell is happening